explainer.md

```diff
@@ -238,6 +238,70 @@ While this is often good practice, it only addresses very limited scenarios:
   browser-facing web services. It would not be viable to use different DNS names
   for, say, older and newer browsers.
 
+### Global Timestamp or Version
+
+Rather than trust-store-specific version numbers, one could imagine a
+Web-PKI-only solution, where the client sends a trust store timestamp or,
+equivalently, a global trust store version number, without reference to
+different trust stores.
+
+In addition to being Web-PKI-specific, this approach already does not match the
+reality of the Web PKI today. A global timestamp or version number presumes
+that all root programs make the same changes in, if not the same time, the
+same order. However, the root programs which make up the Web PKI do not
+coordinate on decisions. Different root programs may make different
+decisions at different times, for a variety of reasons, ranging from:
+
+* Different prioritization and resourcing decisions
+* New CA applications being processed at different times
+* Different decisions on how to remediate untrustworthy CAs
+* Different judgements on what policies best meet their respective users' needs
+
+For example, some root programs adopted a
+[Certificate Transparency](https://www.rfc-editor.org/rfc/rfc6962.html)
+requirement in as early as 2018, while others have not, as of writing in 2024.
```

Comment:
It might be helpful to slightly expand on this point with something to the effect of:

Websites that need to function in both CT-enforcing and non-CT-enforcing user agents are required to include several additional SCT signatures with their certificates, regardless of whether they were needed. With the significant leap in size from ECDSA to current PQC algorithms, this becomes a significant challenge that some user agents may opt to require SCT omission, rather than tacitly ignoring them.

Reply:
Hmm. I guess I was viewing this as a brief illustration of how they diverge today (beyond the obvious one of slightly different trusted CAs), with the impact to PQC the next paragraph. Talking about this here doesn't quite flow, but maybe there's another way to organize it?

The cost of sending stuff that only some clients care about is roughly covered in the section about cross-signing, though I think that needs a bit more text. SCTs are also an example of this, but maybe less directly to the point since we haven't really discussed CT in the context of trust expressions much yet.

Could also just delete this paragraph if it's mostly confusing.

Reply:
Yeah, that's fair about it being better suited in another section, but I think it's worth expanding upon somewhere, especially in light of PQC sig sizes adding the risk that relying parties would overlook their presence as they do today. I'm good with the changes.